### ISSUE TITLE
Updating course description for a specific location (RailsBridge NYC)

### ISSUE THREAD

**Issue by javierjulio** [railsbridge/bridge_troll#526]
I'm new to BridgeTroll so I'm not sure where or how best to do this. For RailsBridge NYC we have an issue where the course description contains a URL to docs.railsbridge.org but we use docs.railsbridgenyc.org. Seems we don't have any capability to change that. It's causing us a lot of grief as some students end up on the wrong URL (e.g. show up late and miss the note). I noticed there is [default seed data and the URL is hardcoded](https://github.com/railsbridge/bridge_troll/blob/master/db/seeds/course_populator.rb#L31) there. Anyway we can edit this but just for NYC based RailsBridge? If this would require a PR I'm happy to contribute the time for it. I wanted to ask as I wasn't really sure where to get started. It seemed odd to me that this isn't something that can be changed through an admin. But perhaps it can be, I'm just unaware. Trying to help our RB NYC organizers.

**Comment by lilliealbert** [comment]
@javierjulio I would actually want to take this as an opportunity to see if we could merge any of the improvements that the NYC fork has back into the main docs repo. Could you tag in anyone from NYC who would be good to have in that conversation?

**Comment by javierjulio** [comment]
@lilliealbert that's great because before posting this we were talking about that on the RB NYC slack. They want to do the same thing, get our docs fork changes back into the main repo. I've asked some on the team to comment here. I'd be happy to contribute the time to take care of any merge conflicts. I'd also like to contribute some other docs changes too and can do that as separate PRs.

**Comment by lilliealbert** [comment]
That would be great!

Looking at the fork, it seems like it's a fork of RailsBridge Boston's fork, which itself is very diverged from the main repo. So we'll probably want to be pretty specific about what y'all find most valuable from the fork, since it's pretty long-lived at this point.

**Comment by javierjulio** [comment]
@lilliealbert yes finding out now that its probably way way off whatever we decide to do. We'll see if we can figure that out. Although this doesn't change anything for us in regards to the link. Is there something we can do temporarily to either not have that URL hardcoded in the course description or have a way to replace it for certain regions?

**Comment by qiheme** [comment]
Hi guys!! Continuing off of what @javierjulio mentioned, we want to begin working some of the changes back into the main Docs repo. We are running a fork of a fork, so it seems like a fresh PR with the changes is the way to go. We were thinking that the VM installfest would be an alternative to the traditional installfest, with some supplemental instruction along the way if you were using a VM install. We also have some changes to the actual curriculum that were incorporated, which we would want to address in separate PRs. Let us know how that sounds @lilliealbert and we can get the ball rolling!

**Comment by lilliealbert** [comment]
I'd be interested in a PR that grabbed just the VM stuff and offered it as a separate Installfest option at the top level of the docs — so maybe "Installfest - Native Install" and "Installfest - Virtual Machine" (or call it something else). I'd like to prevent the confusion of having two separate paths within the current main Installfest, since there are already a billion branching points with all the OSs.

I think it might make sense to try to make one uber-commit with the VM installfest instructions, attribute it to RailsBridge Boston & NYC, and have a PR of just that, rather than try to deal with the git history.

In terms of the curriculum itself, separate PRs sounds great!

If we can do this instead of customizing the curriculum link in the app, that would be 💯 🌟 🎉 ✨ 😍
